hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -2940,6 +2940,31 @@ public List<RepeatedOmKeyInfo> listTrash(String volumeName,
     }
   }
 
+  @Override
+  public SnapshotInfo getSnapshotInfo(String volumeName, String bucketName,
+                                      String snapshotName) throws IOException {
+    metrics.incNumSnapshotInfos();
+    Map<String, String> auditMap = buildAuditMap(volumeName);
+    auditMap.put(OzoneConsts.BUCKET, bucketName);
+    try {
+      if (isAclEnabled) {
+        omMetadataReader.checkAcls(ResourceType.BUCKET, StoreType.OZONE,
+            ACLType.LIST, volumeName, bucketName, null);
```

Comment:
I don't think LIST would be the right ACL check here. It should be `ACLType.READ` similar to other info commands.

Reply:
You are right. Copy-paste bug.

Thanks for the catch.